docs/faststore/docs/getting-started/1-onboarding/dashboard.mdx

```diff
@@ -102,20 +102,39 @@ Once you've set the pages for testing and after a new deployment, go to the **Ov
+The **Settings** tab is where you manage the following configurations:
 
-![settings-webops](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/settings-webops.png)
+- [**Production domain**](#production-domain): The primary URL customers use to access the live store.
+- [**Secrets**](#secrets): Sensitive information securely managed during the FastStore project deployment process.
+
+![settings-webops](https://vtexhelp.vtexassets.com/assets/docs/src/secrets-webops___2869e469b5a613f389f19cc555f4c3c5.png) 
 
 ### Production domain
 
-In the **Production domain** section, choose the store domain from the dropdown list in the **Host** field and click `Add` to set up the primary domain of your store.
+As part of the go-live process, you must define your store’s production domain.
+
+Before adding the domain, make sure the [external DNS settings](https://developers.vtex.com/docs/guides/faststore/go-live-1-configuring-external-dns) are correctly configured in your VTEX account settings.
+
+In the **Production domain** section, select the store domain from the dropdown list in the **Host** field and click `Add` to set up your store's primary domain.
 
-### Current domains
+#### Current domains
 
-The **Current Domains** section displays the default and primary domains that have been configured for your store. This section serves as a reference point to ensure that your domains are correctly set up in WebOps. By checking this list, you can confirm whether your domains have been successfully added and are ready for use in the FastStore environment. 
+The **Current Domains** section displays the default and primary domains configured for your store. This section is a reference point to ensure your domains are correctly set up in WebOps. By checking this list, you can confirm whether your domains have been successfully added and are ready for use in the FastStore environment. 
 
 If your domain isn't listed, it may indicate an issue with the configuration or DNS settings that need to be addressed before proceeding with the go-live.
 
+### Secrets
+
+Secrets, like API keys and passwords, need secure management during FastStore deployment. Standardizing secrets management via WebOps enhances security and consistency across deployment providers.
+
+In the **Secrets** section, define the **Key** and **Value** in their respective fields and click `Add` to set up the secret.
+
+#### Current keys
+
+The **Current keys** section lists all secrets configured for your store. This list shows each secret’s key (the identifier), while the value is hidden by default for security reasons. You can toggle the visibility of a secret’s value if needed.
+
+>ℹ Learn more about Secrets in [Managing secrets](https://developers.vtex.com/docs/guides/faststore/security-managing-secrets).
```

Comment:
Acho que esse callout pode vir antes. Como na seção Secrets já estamos falando disso, faz mais sentido já ter algo ali no início, na linha 31 por exemplo.